pkg/reconcile/pipeline/infinispan/pipeline/pipeline.go

```diff
@@ -222,6 +222,7 @@ func (b *builder) Build() pipeline.Pipeline {
 	handlers.Add(
 		manage.ConsoleUrl,
 	)
+	handlers.AddFeatureSpecific(i.IsServiceMonitorEnabled(), provision.ServiceMonitor)
```

Comment:
This should be removed as we already add this function to the pipeline  on L193.

Reply:
removed the line